docs/LV_client/shared_libraries/mono_radar_plots_with_KF.md

```diff
@@ -0,0 +1,24 @@
+# mono_radar_plots_with_KF.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_radar_plots_with_KF.png"/>
+</p>
+
+### Description
+
+Tool to plot the range, aoa and velocity to compare the outputs of the mono_radar.vi and the mono_ground_radar.vi for a specific Tracking element on the list of target of the ground truth radar sensor. This tool also adds a Kalman Filter to the output of the  mono_radar.vi to smooth the aoa value. 
+
```

Comment:
This part is a little hard to read, could be because I am not familiar with the terminology though. "for a specific Tracking element on the list of target of the ground truth radar sensor. " Could rewrite as "for a specific tracking element in the list, "name of list." Also, should tracking capitalized?

Reply:
I changed it a little bit. Is it better?

Reply:
Remove "Too to" and start the sentence with "Compares" instead. Much easier to read, :)

Reply:
Ups that meant to be tools, but I agree that that "Compares..." is better.